### PR TITLE
Display storage account information in documentation

### PR DESCRIPTION
**Pull request by fashaikh**
Needs to go in together with this PR : https://github.com/projectkudu/AzureFunctionsPortal/pull/592/

### CODE REVIEW

File: Documentation/blobIn.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+#### Connecting to your Storage Account
+
+* Download Storage explorer from here: http://storageexplorer.com/ 
+* Connect using these credentials:
+* Account Name: `{{accountName}}`
+* Account Key: `{{accountKey}}`
```
Comment (fashaikh) [inline Documentation/blobIn.md]: Hmm okay. Will experiment with hiding it

Comment (fashaikh) [inline Documentation/blobIn.md]: BTW I noticed we do display github secret in plain text
image . Still dont have a good answer on hiding this with markdown. Working on it

Comment (mathewc) [inline Documentation/blobIn.md]: Github secrets, WebHook/Http security codes, etc. we do show in plain text you're right. I agree that that is a slight discrepancy. In my mind showing the full storage credentials seems worse though.

Comment (fashaikh) [inline Documentation/blobIn.md]: I'll close this PR. The new change to hide secrets by default will directly be made in the AzureFunctionsPortal, and wont need any changes here.
Adding custom html on the fly is apparently not supported by angular.

**Pull request closed without merging**